testsuite/tests/native-debugger/sanitize.awk

```diff
@@ -31,6 +31,19 @@
     # Replace comments with blank comments
     gsub(/; [a-zA-Z0-9._+ ]+/, ";")
 
+    # Replace line numbers in runtime files - one rule for lldb, one for gdb
+    # (it would be better to only match on runtime/*.c, but gsub does not
+    # handle ERE)
```

Comment:
out of curiosity, what's ERE?

Reply:
Extended Regular Expressions.

I wanted to use something like `(runtime/.*.c):[0-9]+` and replace with `\1:XX` so that only runtime files match.